app/src/main/res/layout/fragment_view_post.xml

```diff
@@ -9,7 +9,7 @@
     <!-- POST UI START -->
     <TextView
         android:id="@+id/post_content"
-        android:layout_width="346dp"
+        android:layout_width="0dp"
```

Comment:
I can't view the UI at the moment. But was the layout switched/changed?

Reply:
Hardcoding the width does not work for different screen sizes, by setting it to `match_constraint` it will automatically resize the text box to the width of the screen. on my devices the way it was right now all of the content of the post message goes off the edge of the screen and is cut off.